app/controllers/report_controller.rb

```diff
@@ -687,7 +687,7 @@ def replace_right_cell(options = {})  # :replace_trees key can be an array of tr
     partial = options[:partial] ? options[:partial] : set_partial_name
     unless @in_a_form
       c_tb = build_toolbar(center_toolbar_filename)
-      h_tb = build_toolbar("x_history_tb")
+      h_tb = build_toolbar("x_history_tb") unless (x_active_tree == "export_tree" || params[:id] == "export_accord")
```

Comment:
@GregP please change check to ` unless x_active_tree == :export_tree ` that should be good enough, value of x_active_tree is a symbol not string